### PR TITLE
Make navigation tab sticky

### PR DESCRIPTION
**Pull request by pazembrz**
ref: inspirehep/inspirehep#1226

### CODE REVIEW

File: src/json-editor.component.scss [hunk 1]
```diff
@@ -564,4 +564,12 @@ complex-list-field {
   add-field-dropdown {
     display: none;
   }
+}
+
+// Make nav tabs sticky
+.nav-tabs{
```
Comment (drjova) [inline src/json-editor.component.scss:570]:
```suggestion
.nav-tabs {
```

File: src/json-editor.component.scss [hunk 2]
```diff
@@ -564,4 +564,12 @@ complex-list-field {
...
+    position: sticky;
+    top: 0;
+    background: white;
+    z-index: 1;
```
Comment (drjova) [inline src/json-editor.component.scss:574]: do we need the `z-index`?

**Comment by drjova** [comment]
@pazembrz could you please attach a screenshot how it looks? Also please remember you will need to bump it on `record-editor` after is merged.

File: src/json-editor.component.scss (hunk 2 above)
Comment (pazembrz) [reply src/json-editor.component.scss:574]: Without it Text from editor is overlapping with this bar
editor_example

**Comment by pazembrz** [comment]
> @pazembrz could you please attach a screenshot how it looks? 
Editor_1

File: src/json-editor.component.scss (hunk 1 above)
Comment (pazembrz) [reply src/json-editor.component.scss]: :+1:

**Review by drjova — APPROVED** [review]

**Pull request merged**